cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/data_explorer/sidebar.spec.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+
+describe('discover sidebar', () => {
+  before(() => {
+    cy.setAdvancedSetting({
```

Comment:
Reset to default after the test

Reply:
the default is `null`, do we have to set it back to `null`?

Reply:
Oh then no worries. But in that case why is this needed?

Reply:
Oh then nvm :)

Reply:
> Reset to default after the test

+1, yes. The value will be null when a user unsets a setting that is default undefined (or null)